src/wasm/literal.cpp

```diff
@@ -254,6 +255,68 @@ Literal Literal::convertUToF64() const {
   WASM_UNREACHABLE();
 }
 
+template <typename F>
```

Comment:
we more commonly have `template<` (without a space), by a factor of 5. please change it here. we should also fix the other places, i'll look into that.